app/controllers/publishers/sessions_controller.rb

```diff
@@ -1,12 +1,37 @@
-class Publishers::SessionsController < Publishers::BaseController
-  protect_from_forgery with: :null_session, only: %i[destroy]
+class Publishers::SessionsController < Devise::SessionsController
+  PUBLISHER_SESSION_KEYS = %i[
+    publisher_id_token
+    publisher_oid
+    publisher_multiple_organisations
+    organisation_urn
+    organisation_uid
+    organisation_la_code
+  ].freeze
 
-  skip_before_action :authenticate_publisher!, only: %i[destroy]
-  skip_before_action :update_publisher_last_activity_at, only: %i[destroy]
-  skip_before_action :check_session, only: %i[destroy]
-  skip_before_action :check_terms_and_conditions, only: %i[destroy]
+  def new; end
```

Comment:
Don't think we need this empty action (rails will just figure it out if the template is there)